students/etwum/lesson03/slicing.py

```diff
@@ -0,0 +1,147 @@
+
+def exchange_first_last(seq):
+    # exchanges the first/last items in a sequence then copies the new sequence into a new string, list or tuple
+
+    # performs this part of the if statement if the sequence is of type string
+    if type(seq) is str:
+        y = seq[-1] + seq[1:-1] + seq[0]
+        new_string = y
+        return new_string
```

Comment:
you can just do `return seq[-1] + seq[1:-1] + seq[0]`, no need for extra variables